### ISSUE TITLE
Development - mkvirtualenv: command not found

### ISSUE THREAD

**Issue by SteveAmor** [RPi-Distro/python-gpiozero#630]
In the docs, section 11.1. (Development installation) details the process to install a virtual Python environment.  However, after installing the Python packages/modules, the command ```mkvirtualenv -p /usr/bin/python3 python-gpiozero``` returns ```-bash: mkvirtualenv: command not found```. 

Tested with both apt and apt-get on new install of Stretch Lite (after a sudo apt-get update).

**Comment by SteveAmor** [comment]
Just been looking at piwheels and the test and the virtualenv it uses so I'm working on a fix for sectionn 11.1.  Will raise a PR later (I hope).

**Comment by SteveAmor** [comment]
Fixed in #632

**Comment by SteveAmor** [comment]
According to the official documentation (https://virtualenvwrapper.readthedocs.io/en/latest/install.html) after creating a virtualenv directory using ```mkdir ~/.virtualenvs``` the following should be added to .bashrc:
```
export WORKON_HOME=/home/pi/.virtualenvs
source /usr/share/virtualenvwrapper/virtualenvwrapper.sh
````
Then run ```source ~/.bashrc```

**Comment by waveform80** [comment]
Fixed in 3685ffb72a1ffb70bc18a1fd62169b442adf0999 - remember to close when docs-tidy-up PR is merged
